From 1ed96a4240d3ef1b8fbd4fca0c86a84df25426f0 Mon Sep 17 00:00:00 2001
From: Sanyog Kale <skale@habana.ai>
Date: Fri, 11 Sep 2020 22:59:05 +0300
Subject: [PATCH] [SW-17444] preboot: goya2_arc: increase stack size

Since fit verification support is been added in preboot which will
also be used by goya2 arc to enable secure boot flow, this patch
increases stack size inline with gaudi preboot stack size

Change-Id: I58ff6e90be3ba0328996edd3494d3fbbeb8ee358
---

diff --git a/goya2_arc_gnu/preboot.lds.S b/goya2_arc_gnu/preboot.lds.S
index 53b1fb1..740f036 100644
--- a/goya2_arc_gnu/preboot.lds.S
+++ b/goya2_arc_gnu/preboot.lds.S
@@ -32,7 +32,7 @@
 	.stack : {
 		. = ALIGN(8);
 		__stack = . ;
-		. = . + 512 ;
+		. = . + 4096 ;
 		__stack_top = . ;
 	} > .sram
 
